action/rlp_tx.go

```diff
@@ -69,3 +56,24 @@ func generateRlpTx(act rlpTransaction) (*types.Transaction, error) {
 	}
 	return types.NewContractCreation(act.Nonce(), act.Amount(), act.GasLimit(), act.GasPrice(), act.Payload()), nil
 }
+
+func signRlpTx(tx rlpTransaction, chainID uint32, sig []byte) (*types.Transaction, error) {
```

Comment:
a better name is `reconstructSignedRlpTxFromSig`
there is no private key, so it is not `signRlpTx`